rllib/examples/centralized_critic.py

```diff
@@ -200,29 +176,47 @@ def central_vf_stats(policy, train_batch, grads):
     }
 
 
-CCPPOTFPolicy = PPOTFPolicy.with_updates(
-    name="CCPPOTFPolicy",
-    postprocess_fn=centralized_critic_postprocessing,
-    loss_fn=loss_with_central_critic,
-    before_loss_init=setup_tf_mixins,
-    grad_stats_fn=central_vf_stats,
-    mixins=[
-        LearningRateSchedule,
-        EntropyCoeffSchedule,
-        KLCoeffMixin,
-        CentralizedValueMixin,
-    ],
-)
+def get_ccppo_policy(base):
+    class CCPPOTFPolicy(CentralizedValueMixin, base):
+        def __init__(self, observation_space, action_space, config):
+            base.__init__(self, observation_space, action_space, config)
+            CentralizedValueMixin.__init__(self)
+
+        @override(base)
+        def loss(self, model, dist_class, train_batch):
+            return loss_with_central_critic(
```

Comment:
Can we add a comment here about the `super()` "trick"? Need to call the super's (actual) loss function inside the central one defined above?

Reply:
yeah, that's right.
added a comment.